### ISSUE TITLE
The field "wkt" is being treated as required in "areas.txt" which does not meet GTFS.org spec

### ISSUE THREAD

**Issue by briandonahue** [OneBusAway/onebusaway-gtfs-modules#210]
**Summary:** 

The field `wkt` is being treated as a required field in `areas.txt` which is causing validation errors such as:
https://github.com/MobilityData/gtfs-realtime-validator/issues/92

The `wkt` field seems to be an [experimental spec](https://trilliumtransit.com/gtfs/reference/#areas) and not part of the [GTFS.org spec](https://gtfs.org/schedule/reference/#areastxt).  

**Steps to reproduce:** 

You can follow the steps in the [original issue](https://github.com/MobilityData/gtfs-realtime-validator/issues/92) which uses the `CsvEntityReader` to read the `areas.txt` file and throws an exception:
```
org.onebusaway.csv_entities.exceptions.CsvEntityIOException: io error: entityType=org.onebusaway.gtfs.model.Area path=areas.txt lineNumber=2
```
When the file follows the GTFS.org spec with 2 fields, `area_id, area_name`

**Expected behavior:** 

Not throw when the `areas.txt` meets the GTFS.org spec

**Observed behavior:** 

throws an exception:
```
org.onebusaway.csv_entities.exceptions.CsvEntityIOException: io error: entityType=org.onebusaway.gtfs.model.Area path=areas.txt lineNumber=2
```

**Platform:** 

java 17 2021-09-14 LTS
onebusaway-csv-entities-1.1.6
onebusaway-gtfs-1.3.87

**Comment by sheldonabrown** [comment]
@briandonahue Thanks for the PR.  Its released now as 1.3.118

**Comment by briandonahue** [comment]
Fantastic, thanks!
